playbooks/openshift-monitoring-blackbox/private/components.yaml

```diff
@@ -0,0 +1,6 @@
+# To include apps to be installed into the blackbox project, import their
+# installation playbooks here.
+
+# Example:
+#
+# - import_playbook: ../../openshift-monitoring-health-lifecycle/private/config.yml
```

Comment:
@jhadvig @bparees this would be the place to update to install your test app for monitoring

Reply:
so we should import them as separate playbooks...
Also do we expect to install all the monitoring components by default ? or it should be configurable ?

Reply:
Separate playbooks, yes. The monitoring platform role itself (openshift_cluster_monitoring_operator) and blackbox test role (openshift_cluster_monitoring_blackbox) are off by default and each have 1 variable to control installation/removal of the stuff.

So we only need 1 variable to control installation of *all the blackbox tests*. If somebody wants to run the individual playbooks against a cluster for ad-hoc development/testing that's fine, but there won't be individual variables for each blackbox test app.

Make sense?

Reply:
@ironcladlou i've added a link to this commit to the doc.